### PR TITLE
User interface

### PR DESCRIPTION
**Pull request by jcromerohdz**
Building the CLI for the tic tac toe project

### CODE REVIEW

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Requires Changes

Hi team, good job so far! :slightly_smiling_face: 

However, there are few updates we need to make.


- [x] The game should ask the player to select from available moves, **```(1 - 9)```**.

- [ ] The player should be notified if a move is an invalid move.

- [ ] Also, when a player wins or loses a game, there should be a notification.

- [ ] There should be notification if the outcome of the game is a tie/draw.

### Make these updates and submit another form for a code review.

#### ```Have a great day``` :slightly_smiling_face: 

@ Uchechukwu Henry Inyama

File: bin/main.rb [hunk 1]
```diff
@@ -2,4 +2,120 @@
...
+        system("clear")
+      when 2
+        if @board[(move.to_i-1)] != "X" || @board[(move.to_i-1)] != "O"
+          @moves+=1
+          @board[(move.to_i-1)]="O";
+        else
+          puts "invalid choice"
+          player_in(2)
+        end
+        system("clear")
```
Comment (maelfosso) [inline bin/main.rb:85]: Not working
```ruby
 O |   |  
-----------
   |   |  
-----------
   |   |  
 
1 | 2 | 3
4 | 5 | 6
7 | 8 | 9
moves: 2
 
 
 O |   |  
-----------
   |   |  
-----------
   |   |  
 
1 | 2 | 3
4 | 5 | 6
7 | 8 | 9
moves: 2
 
Mael select the position to play: 
1

 X |   |  
-----------
   |   |  
-----------
   |   |  
 
1 | 2 | 3
4 | 5 | 6
7 | 8 | 9
moves: 3
 
Elvis select the position to play: 
```

File: bin/main.rb [hunk 2]
```diff
@@ -2,4 +2,120 @@
...
+    case player
+      when 1
+        if @board[(move.to_i-1)] != "X" || @board[(move.to_i-1)] != "O"
+          @moves+=1
+          @board[(move.to_i-1)]="X"
+        else
+          puts "invalid choice"
+          player_in(1)
+        end
+        system("clear")
```
Comment (maelfosso) [inline bin/main.rb:76]: **`[OPTIONAL]`**
I think you should remove it. It's better to leave the message get display one after another.

File: bin/main.rb [hunk 3]
```diff
@@ -2,4 +2,120 @@
...
+    input=nil
+    until input
+      input=gets.chomp.to_i
+      case input
+        when 1..9
+          #input range
+        else
+          puts "invalid choice"
+          input = nil
+      end
```
Comment (maelfosso) [inline bin/main.rb:46]: Please, use 
```ruby
if Array(1..9).include? input
...
end
```

**Review by maelfosso — CHANGES_REQUESTED** [review]
# Not approved. Just one bug to fix

Hello coders,

It's not OK yet :disappointed: 
Please, read the comments below to learn more about it.
Kindly, make the changes and submit a new code review request.

**Mael FOSSO**
[Github](https://github.com) | [Twitter](https://twitter.com)

**Review by uche-inyama — APPROVED** [review]
## Hi team, 

Implementing the flow of the game logic as required in milestone 2 (user-interface) is working properly; using get.chomp was also properly implemented.

Go ahead and merge the milestone - user interface branch, with the master branch
#### ``` Have a nice day``` :slightly_smiling_face: 

@ Uchechukwu Henry Inyama

**Pull request merged**